pkg/http/http.go

```diff
@@ -0,0 +1,33 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/go-logr/logr"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/tinkerbell/pbnj/server/grpcsvr/taskrunner"
+)
+
+var (
+	logger     logr.Logger
```

Comment:
any chance we could make this (and the taskrunner) http middleware instead of global vars? global vars can be difficult to test and dont always play nice with serving concurrent/multiple requests. Feels like it might affect the count. The logger and taskRunner could potential go into a struct and then `handle...` functions could be methods off that struct. just an idea.

Reply:
yeah, that's definitely how it should be done. I might have been a little lazy. I'll update that.